CS3230Project/DAL/Nurses/NurseDal.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using CS3230Project.Model.Users;
+using MySql.Data.MySqlClient;
+
+namespace CS3230Project.DAL.Nurses
+{
+    /// <summary>
+    ///   The data access layer for the nurses
+    /// </summary>
+    public static class NurseDal
+    {
+
+        /// <summary>
+        /// Gets the nurses.
+        ///
+        /// Precondition: none
+        /// Post-condition: none
+        /// </summary>
+        /// <returns>
+        ///   A list of all the nurses
+        /// </returns>
+        public static List<Nurse> GetNurses()
+        {
+            using var connection = new MySqlConnection(Connection.ConnectionString);
+            connection.Open();
+            const string query = "select * from nurses";
+            using var command = new MySqlCommand(query, connection);
+
+            return NurseDal.createNurses(command);
+        }
+
+        /// <summary>
+        /// Gets the nurse by identifier.
+        ///
+        /// Precondition: none
+        /// Post-condition: none
+        /// </summary>
+        /// <param name="ID">The identifier.</param>
+        /// <returns>
+        ///   the nurse with the provided ID
+        /// </returns>
+        /// <exception cref="System.ArgumentException"></exception>
+        public static Nurse GetNurseByID(int ID)
```

Comment:
There are some ordinals that are not used. They should be removed

Reply:
I will fix this. I was waiting on the merge.

Reply:
Fixed. 

Also, the GetNurseByID() method is not currently being used. I was thinking wrong while implementing something else, and by the time I had realized, I was done with the query and method. It can be removed, but I left it for now just in case we could use it in the near future.